canaries/2D_image_stream.py

```diff
@@ -9,12 +9,25 @@
 from viam.components.camera import Camera
 from viam.robot.client import RobotClient
 from viam.rpc.dial import Credentials, DialOptions
+from viam.components.camera.client import CameraMimeType
 
 import logging
+import requests
+import os
 
+url = os.getenv('MY_SLACK_WEBHOOK')
+arg = int(os.getenv('ARG'))
```

Comment:
[opt] Alternatively you could use [command line args](https://www.tutorialspoint.com/python/python_command_line_arguments.htm) but this is fine.